elastic/kibana
# [Security Solution] fix alert dashboard filters

## Summary
This PR fixes the following issue:
https://github.com/elastic/kibana/issues/177906

Applied the fix suggested by Xavier, and it seems to resolve the issue.

To test:
1. On current main, generate large volume of alerts, switch to 100 rows per page, then go to the last page.
2. Filter in on any column, eg. @timestamp. You should see the exact value applied to top filters.
